atomic_reactor/plugins/pre_pyrpkg_fetch_artefacts.py

```diff
@@ -11,11 +11,18 @@
 This plugin should do it.
 """
 import os
+import re
+import shutil
 import subprocess
+from pathlib import Path
+from typing import List, final
 
+from atomic_reactor.dirs import BuildDir
 from atomic_reactor.plugin import PreBuildPlugin
 from atomic_reactor.constants import PLUGIN_DISTGIT_FETCH_KEY
 
+EXPLODED_SOURCES_FILE: final = 'source-repos'
```

Comment:
```suggestion
EXPLODED_SOURCES_FILE: Final = 'source-repos'
```
According to https://www.python.org/dev/peps/pep-0591/#motivation, `final` is a decorator and `Final` is the type annotation.

Reply:
(nice find on `Final` though, I didn't know it existed :+1:)

Reply:
Good catch! Thanks :smiley: